accounts.go

```diff
@@ -30,6 +30,10 @@ import (
 	"unsafe"
 )
 
+type LikeAccounts interface {
```

Comment:
Would be better to expose a `avl.Snapshottable` interface in the `avl` pkg.

Reply:
in go usually you name interfaces in the form 'er, like Snapshotter :D

Reply:
True - actually this interface in particular doesn't appear to be needed from what I see. The point of this interface is to allow for mocking in `tx_applier_test.go`, when what could be done instead is just struct embedding over `avl.Tree` and overriding the Snapshot() method.